openwisp-config/files/openwisp.agent

```diff
@@ -854,12 +854,12 @@ if [ ! -f "$BOOTUP" ]; then
 	# actions to do only after agent start, not after registration or agent restart
 	if [ "$BOOTUP_DELAY" -ne "0" ]; then
 		# random bootup delay between 0..300 seconds
-		# shellcheck disable=SC3028
-		DELAY="$((RANDOM % BOOTUP_DELAY))"
+		SEED=$(head /dev/urandom | tr -dc "0123456789" | head -c9)
```

Comment:
I think this can be done in the lua script, otherwise the script is pretty much useless and doesn't really give a random number if the seed is the same